store.cpp

```diff
@@ -1,8 +1,168 @@
 #include <bits/stdc++.h>
 using namespace std;
-#include "weapon-generator.cpp"
+#include "item-generator.cpp"
+#include "main-character.cpp"
 
 item itens[3];
-void seeStore();
-void exitStore();
-void buyItem();
\ No newline at end of file
+bool bought[3];
+const int potion_price = 5;
+int actual_potion_price;
+const string potion_text = "Poção (Recupera X de vida)";
+const string store_welcome_text = "BEM VINDO MEU PARCEIRO\n";
+const string already_bought_text = "Você já comprou esse item, escolha novamente\n";
+const string exit_text = "Xau xau meu parceiro\n";
+const string not_enought_coins_text = "Sinto muito, você não me parece ter dinheiro suficiente para comprar isso\n";
+const string invalid_option_text= "Opção Inválida, escolha novamente\n";
+
+void see_store(main_character character) {
+    calculate_actual_potion_price(character);
+    printf("%s",store_welcome_text );
```

Comment:
you have to keep a pattern, there is no space between the variable and the comma here, but there is between the variable and )